src/syslog_mcp/prompts/__init__.py

```
@@ -0,0 +1,8 @@
+"""
+Syslog MCP Server - Prompts Module
+
+This module contains reusable prompt templates that guide users through
+structured syslog analysis workflows using our MCP tools.
+"""
+
+# Empty init file for now - prompts will be registered in individual modules
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)